### ISSUE TITLE
RankESRGAN underperform as expected

### ISSUE THREAD

**Issue by ThreeSevenGO** [WenlongZhang0724/RankSRGAN#4]
Thank you for your amazing work!
I replace the base model as RRDB_net, with using the original Ranker model. The SR results are disappointing. It's like even GAN is not working that the SR results don't recovery details.
I guess you did this experiment, too. Did you get the same result? Maybe I should retrain the Ranker model, do I ?

Best wish.

**Comment by WenlongZhang0724** [comment]
Sorry to reply to you so late. I have tested the RRDB net and the SR results are OK. Could you tell me the specific setting and some experimental results, like PSNR/NIQE curve and some visual results?  You also could send the results to my email (zhangwenlong0724@gmail.com)

**Comment by zdyshine** [comment]
Thank you for your amazing work!
Could you please tell me how to set the weight of the loss function, if I use RRDB_Net ? Is that ok by default?

**Comment by WenlongZhang0724** [comment]
Yeah, the weight of loss function is set by default.

**Comment by linxiaojinjin** [comment]
Thank you for your amazing work!
Where can I download the pre training model of esrgan？
